### PR TITLE
fix: sbatch stderr parsing #2

### PR DESCRIPTION
**Pull request by cmeesters**
next attempt to fix the behaviour described in #161 

## Summary by CodeRabbit

- **Bug Fixes**
	- Improved error handling for SLURM job submissions, providing clearer indications of submission failures.
	- Enhanced error reporting with detailed messages for job submission failures.

- **Chores**
	- Minor adjustments to comments and formatting for better clarity.

### CODE REVIEW

**Comment by freekvh** [comment]
I'm happy to report that it works as expected!

**Comment by cmeesters** [comment]
@fgvieira Hoi Filipe, time for another go? ;-)

**Comment by fgvieira** [comment]
Sure! :)

The two error messages are identical and that might make it a bit hard to figure out why it failed.
Could we make one of the messages more specific?

**Comment by fgvieira** [comment]
I did not meant merge them, but maybe have different error messages specifying the different nature of the errors.

**Comment by cmeesters** [comment]
yeah, fine. Inserting a clarification and distinction is just good.

**Comment by cmeesters** [comment]
@fgvieira I truncated the error string again - otherwise the linter will fail. Breaking the string in a multiline string yielded ugly output. The current string is a compromise.

File: snakemake_executor_plugin_slurm/__init__.py [hunk 1]
```diff
@@ -231,9 +231,10 @@ def run_job(self, job: JobExecutorInterface):
                 )
         except subprocess.CalledProcessError as e:
             raise WorkflowError(
-                f"SLURM job submission failed. The error message was {e.output}"
+                f"sbatch command failure. The error message was {e.output}."
```
Comment (fgvieira) [inline snakemake_executor_plugin_slurm/__init__.py:234]:
```suggestion
                f"SLURM sbatch failed. The error message was {e.output}"
```

**Review by fgvieira — APPROVED** [review]

**Pull request merged**